### DISCUSSION TITLE
Taylor Otwell pull request framework larvael

### DISCUSSION THREAD

**Discussion by No-Dively** [laravel/framework#43257] [General] [answered]
Hi
I have question.Why Taylor Otwell closed pull request without say something?

**Comment by No-Dively** [comment]
@taylorotwell

**Comment by erikn69** [comment, marked as answer]
He almost always post this

>Thanks for your pull request to Laravel!
>
>Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include.
>
>If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions!
>
>If you feel absolutely certain that this code corrects a bug in the framework, please "@" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

Maybe too much to review, or he don't like big changes from unknown people

**Reply by Sophist-UK** [reply]
To be honest I can understand his perspective, but as always there are downsides to whatever choice he makes, in this case the creation of a **LOT** of small, sometimes unsupported, packages, giving users (like us) a lot of chaff to sort through to find the gems we might like. Too much choice and a highly fragmented set of enhancements can be just as bad as a bloated code base.

But Laravel is @taylorotwell 's project and so entirely his choice of approach.
